mungegithub/mungers/approvers/approvers_test.go

```diff
@@ -594,13 +596,15 @@ func TestGetMessageAllApproved(t *testing.T) {
 			}),
 		},
 	)
-	ap.AddApprover("Alice", "REFERENCE")
-	ap.AddLGTMer("Bill", "REFERENCE")
+	ap.AddApprover("Alice", "REFERENCE", false)
+	ap.AddLGTMer("Bill", "REFERENCE", false)
 
-	want := `[APPROVALNOTIFIER] This PR is **APPROVED**
+	want := `[APPROVALNOTIFIER] This PR is **NOT APPROVED**
 
 This pull-request has been approved by: *<a href="REFERENCE" title="Approved">Alice</a>*, *<a href="REFERENCE" title="LGTM">Bill</a>*
 
+*No associated issue*. Update body to associate one, or get approval with ` + "`/approve no-issue`" + `
```

Comment:
Maybe we should clarify this. "Update Pull Request to add a reference to an issue..."